use/2016-01-26-SNP-selection.Rmd

```diff
@@ -211,19 +211,25 @@ ggplot(loc2temp, aes(x = factor(loc7), y = temp)) +
  ylab("Temperature (Centigrade)")
 
 mod <- glm(cbind(loc7, 2 - loc7) ~ temp, family = binomial)
-summary(mod) ## This loci is not significantly correlated to temperature
+summary(mod) ## This locus is not significantly correlated to temperature
  
 ```
 
- 
+Now that we are done with the analysis, we can remove the temporary file that
+*PCAdapt* has created:
```

Comment:
The R package wraps some older C code, so it's used to reading and writing directly from files on disk and using temporary files as storage. I've pointed this out to the author, so hopefully a new version will not use the temporary files. 

My inclination now is to leave this part out and leave the tempfile there.